Jenkinsfile

```diff
@@ -6,12 +6,15 @@ def defineFlavor() {
     }
 
     def branchName = env.BRANCH_NAME
+
     if (branchName == "main") {
-        return 'Internal'
+        return 'Beta'
```

Comment:
i did rename what we call internal (wire beta) to Beta and added internal flavor to replace wire internal wioth greren icon